atomos.framework/src/main/java/org/atomos/framework/base/AtomosCommands.java

```diff
@@ -86,29 +92,57 @@ private String getState(Bundle b) {
 		}
 	}
 
-	public void install(String name, String loaderType, File moduleDir) throws BundleException {
+	@Descriptor("Install a new layer")
+	public void install(@Descriptor("Name of the layer") String name,
+			@Descriptor("LoaderType of the layer [OSGI, SINGLE, MANY]") String loaderType,
+			@Descriptor("Directory where the AtomosBundles are stored") File moduleDir)
```

Comment:
This should read "Directory containing for bundles JARs to install into the layer"

Reply:
done